core/modules/ccontrol/MergingRequester.cc

```diff
@@ -62,12 +63,15 @@ MergingRequester::MergingRequester(
       _tableName(tableName),
```

Comment:
Just upgraded to g++ (GCC) 4.9.2 20150212 (Red Hat 4.9.2-6), and it still doesn't like it.
build/ccontrol/MergingRequester.cc:66: error: no matching function for call to 'std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(<brace-enclosed initializer list>)'